### PR TITLE
improve error reporting for empty levels

### PR DESCRIPTION
**Pull request by mambocab**
This improves the error we saw [here](https://issues.apache.org/jira/browse/CASSANDRA-12275) and provides some debugging output. Doesn't fix the underlying error, though.

### CODE REVIEW

File: offline_tools_test.py [hunk 1]
```diff
@@ -175,11 +175,17 @@ def sstableofflinerelevel_test(self):
 
         # Let's reset all sstables to L0
         debug("Getting initial levels")
-        initial_levels = self.get_levels(node1.run_sstablemetadata(keyspace="keyspace1", column_families=["standard1"]))
+        initial_levels = list(self.get_levels(node1.run_sstablemetadata(keyspace="keyspace1", column_families=["standard1"])))
+        self.assertNotEqual([], initial_levels)
+        debug('Got:')
```
Comment (ptnapoleon) [inline offline_tools_test.py]: Here and below, can you be more verbose than "Got" please

**Comment by mambocab** [comment]
Good idea, thanks. Pushed requested change @ptnapoleon

**Comment by ptnapoleon** [comment]
+1

**Comment by mambocab** [comment]
Thanks, merging.

**Pull request merged**